### PR TITLE
Ubuntu/focal: fix lintian warnings

### PR DESCRIPTION
**Pull request by aciba90**
do not squash

[SC-1154](https://warthogs.atlassian.net/browse/SC-1154)

### CODE REVIEW

**Comment by aciba90** [comment]
Pending: https://github.com/canonical/cloud-init/pull/1566#issuecomment-1177629649

**Comment by aciba90** [comment]
> Pending: [#1566 (comment)](https://github.com/canonical/cloud-init/pull/1566#issuecomment-1177629649)

Done. Ready to be reviewed.

File: debian/control [hunk 1]
```diff
@@ -3,7 +3,7 @@ Section: admin
 Priority: optional
 Homepage: https://cloud-init.io/
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
-Build-Depends: debhelper (>= 9.20160709),
+Build-Depends: debhelper-compat (= 10),
```
Comment (paride) [inline debian/control:6]: I think this should be safe as we're packaging the same codebase to Bionic with compat level 9 (per `ubuntu/bionic branch`). However this requires checking what changes between different compat versions, see `man debhelper` or `man debhelper-compat-upgrade-checklist` if you're running Kinetic.

I had a look and the changeset looks safe (but please double check!), but in principle this is a nontrivial change.

Comment (aciba90) [reply debian/control:6]: I checked the changes superficially, and I did not notice we were hitting a breaking change. `sbuild` runs correctly too. Therefore, I would say it is safe.

**Review by blackboxsw — APPROVED** [review]
- build-package/sbuild success lintian warnings squashed
- travis CI to focal +1
- d/changelog entry format, package versioning and separate commits +1
- will chat about augmented .XX.YY.1 vs ~XX.YY.1 diminished pacakge versioning separately

File: debian/changelog [hunk 2]
```diff
@@ -1,8 +1,11 @@
 cloud-init (22.2-0ubuntu1~20.04.2) UNRELEASED; urgency=medium
 
-  * d/control: add python3-debconf to Depends and Build-Depends
+  * d/control:
+    - add python3-debconf to Depends and Build-Depends
+    - Build-Depends: bump debhelper-compat to v10
```
[the remark this answers is not in the document]
Comment (aciba90) [reply debian/changelog:5]: Yes, I explained it in https://github.com/canonical/cloud-init/pull/1567/commits/0f6db8c5dc51692a808d79b14255692e7c414a59, but I should have been more explicit in the changelog.

**Pull request merged**